packages/manager/apps/dedicated/client/app/dedicatedCloud/terminate/dedicatedCloud-terminate.routes.js

```diff
@@ -1,15 +1,16 @@
-angular.module('App').config(($stateProvider) => {
+export default /* @ngInject */ ($stateProvider) =>
   $stateProvider.state('app.dedicatedClouds.terminate', {
     url: '/terminate',
-    templateUrl: 'dedicatedCloud/terminate/dedicatedCloud-terminate.html',
-    controller: 'DedicatedCloudTerminateCtrl',
-    layout: {
-      name: 'modal',
-      toChilds: true,
-      ignoreChilds: [
-        'app.dedicatedClouds.datacenter',
-        'app.dedicatedClouds.terminate-confirm',
-      ],
+    views: {
+      modal: {
+        component: 'dedicatedCloudTerminate',
+      },
+    },
+    layout: 'modal',
+    resolve: {
+      serviceInfos: /* @ngInject */ ($http, serviceName) =>
+        $http
+          .get(`/dedicatedCloud/${serviceName}/serviceInfos`)
```

Comment:
caching not required?

Reply:
We will cache to potentially remove it as we are most likely to PUT on the same route, not sure it is worth caching